dashboard-app/src/util/dataManipulation/aggregatedToCsv.ts

```diff
@@ -1,15 +1,18 @@
 import csv from 'fast-csv';
 import { AggregatedData } from './logsToAggregatedData';
+import { aggregatedToTableData, abbreviateDateString } from './aggregatedToTableData';
+import { DurationUnitEnum } from '../../types';
+import { Objects } from 'twine-util';
+
+
+// tslint:disable-next-line: max-line-length
+export const aggregatedToCsv = async (data: AggregatedData, unit: DurationUnitEnum): Promise<string> =>
+  new Promise((resolve, reject) => {
+    const tableData = aggregatedToTableData({ data, unit });
```

Comment:
Can we manipulate this data with the functionality that it needs? Seems  counter productive to separate out functionality to then reuse `aggregatedToTableData`

Reply:
I was going for minimal diff, but if you're happy for me to do a bit of re-organisation then sure.

Reply:
Actually turns out I only needed one of the exported util functions, didn't re-organise anything, lol